### DISCUSSION TITLE
Pretrained Densenet models for 3D Mr images

### DISCUSSION THREAD

**Discussion by RRouhi** [Project-MONAI/MONAI#7821] [General]
Hello,

How can I use pre-trained Densenet models for 3D MR image classification using Monai?

Thank you!

**Comment by KumoLiu** [comment]
Hi @RRouhi, thanks for your interest here.

Here are some tutorials you can refer to:
https://github.com/Project-MONAI/tutorials/tree/main/3d_classification
https://github.com/Project-MONAI/tutorials/tree/main/2d_classification
https://github.com/Project-MONAI/tutorials/blob/main/computer_assisted_intervention/endoscopic_inbody_classification.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/self_supervised_pretraining/vit_unetr_ssl/ssl_train.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/self_supervised_pretraining/vit_unetr_ssl/ssl_finetune.ipynb

Thanks.
